### ISSUE TITLE
undelegate stake from myself to myself

### ISSUE THREAD

**Issue by gitusereos** [tarassh/fairy-wallet#136]
Hi, I've just started with EOS, I had some in an exchange and decided to transfer to fairy-wallet for security. I've read the tutorial and decided to stak, so I delegated to myself (I think my name was pre-selected), now I'm trying to undelegate, I did like the step-by-step process in the documentation and waited for the 3 days (did it for four or five times...) but nothing happens! I'm able to complete the transaction, but when I click in the transaction id to chec it I just see a message "Nothing found :(". Anyone knows how to solve it? Thanks!
![Captura de Tela (34)](https://user-images.githubusercontent.com/68715707/88347745-dfc3fe00-cd21-11ea-9846-4f622dfd9c41.png)

**Comment by mind-zipper** [comment]
I am having the same problem trying to unstake my eos.  Sometimes I get transaction successful but transaction id yields "nothing found."  Wait 3 days and....nothing is unstaked.  Other times the transaction fails.  And I have noticed that there is a long communication delay between the nano and the PC when it is asking to confirm the transaction.....weird.  I have tried scatter wallet, fairy wallet, and bloks.io.....all give me the same results.  I have my nano s eos settings, arbitrary data setting to "yes".  Can anyone please help us?

**Comment by mind-zipper** [comment]
Update --  It must have been a communication (signing) issue with the Nano hardware.  I turned off my VPN, my anti-virus and my firewall on my computer (temporarily).  Rebooted my computer, turned on my nanos s with pass phrase, went back into fairy wallet, executed the" unstake/undelegate" transaction.  It executed perfectly after I confirmed on my nano.  I looked at the transaction ID and it said it was "executing.  After about 3 minutes, the transaction had completed.  I went back to the fairy wallet and it stated that my EOS had been unstaked and will be available in 3 days!

**Comment by gitusereos** [comment]
Thanks, I'll try that.

Em ter., 28 de jul. de 2020 às 15:32, mind-zipper <notifications@github.com>
escreveu:

**Comment by mind-zipper** [comment]
Any luck?  People are having these signing issues when using ledger with a Mac.

**Comment by gitusereos** [comment]
Unfortunately, no luck so far... yesterday I was able to see my tokens in
https://eosauthority.com/, and tried to undelegate from there, I got some
issues, but also some transactions completed, but I don't know if it will
be really completed, have to wait the 3 days. I use windows and linux.

Em qui., 30 de jul. de 2020 às 17:18, mind-zipper <notifications@github.com>
escreveu:

**Comment by mind-zipper** [comment]
If you go to bloks.io website, put your account name in the search tab (you don't need your ledger connected), you should see some amount as "refunding".  The refunding amount is what you receive in 3 days.
    On Saturday, August 1, 2020, 05:24:50 AM MST, gitusereos <notifications@github.com> wrote:  
 
 

Unfortunately, no luck so far... yesterday I was able to see my tokens in
https://eosauthority.com/, and tried to undelegate from there, I got some
issues, but also some transactions completed, but I don't know if it will
be really completed, have to wait the 3 days. I use windows and linux.

Em qui., 30 de jul. de 2020 às 17:18, mind-zipper <notifications@github.com>
escreveu:

**Comment by gitusereos** [comment]
I'll try that too...

Em sáb., 1 de ago. de 2020 às 10:09, mind-zipper <notifications@github.com>
escreveu:

> If you go to bloks.io website, put your account name in the search tab
> (you don't need your ledger connected), you should see some amount as
> "refunding".  The refunding amount is what you receive in 3 days.
> On Saturday, August 1, 2020, 05:24:50 AM MST, gitusereos <

**Comment by gitusereos** [comment]
Hello, just to you know, I was able to undelegate and withdraw my eos using
https://eosauthority.com/
https://eostoolkit.io
https://get-scatter.com/

I did a lot of tries but in the and it worked, good luck.

Em dom., 2 de ago. de 2020 às 11:43, Aldair Pinto <aldairsax@gmail.com>
escreveu:

> I'll try that too...
>
> Em sáb., 1 de ago. de 2020 às 10:09, mind-zipper <notifications@github.com>
> escreveu:
>
>> If you go to bloks.io website, put your account name in the search tab
>> (you don't need your ledger connected), you should see some amount as
>> "refunding".  The refunding amount is what you receive in 3 days.
>> On Saturday, August 1, 2020, 05:24:50 AM MST, gitusereos <

**Comment by antoinepouplier** [comment]
I have the same problem, I would like to remove my EOS staked, switch them from CPU to available, but I can't. Would you have a solution for me? 
Thanks in advance,

**Comment by gitusereos** [comment]
Yes, after many trials I was able to unstake almost all of it (you can't
take it all cause some is needed for the transaction). I've used Scatter
wallet in conjunction with my ledger nano and and the website eostolkit.io.
It's not so easy to time the transaction correctly, but I was able to send
it to binance and sell it off! Be aware of scams, never put your seed in
any site or give it to anyone, good luck.

Em terça-feira, 6 de abril de 2021, antoinepouplier <
***@***.***> escreveu:
